nsist/commands.py

```diff
@@ -20,14 +22,20 @@
 
 if __name__ == '__main__':
     from {module} import {func}
-    {func}()
+    sys.exit({func}())
 """
 
+def find_exe(bitness=32):
+    distlib_dir = osp.dirname(distlib.scripts.__file__)
+    return osp.join(distlib_dir, 't%d.exe' % bitness)
```

Comment:
@vsajip is it OK to rely on the location and naming of the exe files like this? I didn't see any better API to locate them. If you'd prefer, I'm happy to contribute the necessary API to distlib and then change Pynsist to use it.

Reply:
`distlib` itself accesses the launchers as `bytes` objects through its own resource-finder API. You can access the paths instead of the bytes using the same API, like this:
```
Python 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 16:02:32) [MSC v.1900 64 bit (AMD64)] on win32 
Type "help", "copyright", "credits" or "license" for more information.
>>> from distlib.resources import finder
>>> distlib_package = 'distlib'  # or 'foo.distlib' if distlib is vendored, for example
>>> finder(distlib_package).find('w64.exe').path  # likewise for t32.exe, t64.exe, w32.exe
'C:\\Users\\Vinay\\Projects\\distlib\\distlib\\w64.exe'
>>>
```

Reply:
Thanks! And you're reasonably confident that the location and the filenames will stay the same?

Reply:
Yes, reasonably confident - no plans to change the location or the names, would certainly involve you, `pip` maintainers and other users in discussion if I was thinking of changing them! They have been that way since initial release, IIRC.

Reply:
Super :+1: